.ebextensions/01migrate.config

```diff
@@ -1,4 +0,0 @@
-container_commands:
```

Comment:
Yeah this is temporary while Frank troubleshoots why our containers are dying when we run `docker exec`.